docs/hyperscript.md

```diff
@@ -539,7 +539,7 @@ Instead, prefer using Javascript expressions such as the ternary operator and Ar
 ```javascript
 // PREFER
 var BetterListComponent = {
-	view: function() {
+	view: function(vnode) {
```

Comment:
Did you forget to migrate this component?

Reply:
No, it's just a fix for an error I spotted (the code that follows refers to the vnode.)

Reply:
I caught the fixed error; I was just referring to the fact it's an object component instead of a closure component.

Hind sight, I probably should've left this comment one line higher.

Reply:
This component has no state - I left stateless components as POJOs (the majority of examples are stateless.)